project.yaml

```diff
@@ -15460,7 +14824,7 @@ actions:
     outputs:
       moderately_sensitive:
         table2: output/venn_vax.csv
-        table2_rounded: output/venn_vax_rounded.csv
+        table2_rounded: output/venn_vax_midpoint6.csv
```

Comment:
Last thing. You haven't updated the Venn script so this is currently mismatched. If you are rerunning, I would update the script. If not, this can be left as `venn_vax_rounded.csv` so that the code works.

Reply:
Indeed it is a mistake, the output name shouldn't be table2! Thanks Venexia!